fsx/README.md

```diff
@@ -0,0 +1,41 @@
+# miniwdl-aws-terraform/fsx
+
+This variant of the [miniwdl-aws-terraform](https://github.com/miniwdl-ext/miniwdl-aws-terraform) configuration deploys [FSx for Lustre](https://aws.amazon.com/fsx/lustre/) (FSxL) instead of [EFS](https://aws.amazon.com/efs/). FSxL has greater upfront costs than EFS, but offers higher throughput scalability needed for large-scale operations.
+
+Compared to the default EFS stack, the key differences here are:
+
+1. FSxL and Batch compute environments are confined to one availability zone
+2. Compute environment for workflow jobs (running miniwdl itself) uses EC2 instead of Fargate
+3. Workflow and task compute environments have an additional cloud-init script that mounts FSxL
+
```

Comment:
Unlike EFS, FSxL doesn't autoscale up or down so you must provision what you will need.